### ISSUE TITLE
Blender Error: No Objects or Images to Bake to

### ISSUE THREAD

**Issue by chrisington** [PlaceholderGames/1stYearHelpDesk#50]
I'm currently following this tutorial:
https://www.youtube.com/watch?v=0LPa5akLdOU

At 17:20 the user selects his high polly model and then his low polly model and uses bake normals to apply his UV Image to create a normal map. 

When I do this process I always get the error "No objects or images to bake to", I've followed this tutorial pretty closely and been stuck on this issue for a while now. I've even tried starting the process over multiple times. 

I also tried using this tutorial: https://www.youtube.com/watch?v=0r-cGjVKvGw but had no luck with this either as the nodes that are created are differenet to the ones that appear when he creates them in the video.

**Comment by DoctorMikeReddy** [comment]
I'll try to have a look tomorrow

**Comment by chrisington** [comment]
Ok, I think I have sorted this out now. I'm not entirely sure what I did differently, but I think it might have been to do with an extra material that had been applied to the objects in Blender. 

After watching videos further in the series, the person in the video recommends creating layers for the couch legs and the base of the couch, so that they don't share the same textures in unreal. This is where I noticed materials had already been created for my model (I'm not sure how, maybe this happened when I was messing about with the nodes from the second video I posted here). When I deleted this layer and tried the baking process again everything was successful: 
![image](https://user-images.githubusercontent.com/43669785/52914421-d4b0a700-32bf-11e9-8b5d-abf1d99047aa.png)
